types/yanzi/presenceMini/uplink.js

```diff
@@ -14,6 +23,23 @@ function consume(event) {
     if (resourceType === "SampleMotion") {
       sample.motion = value;
       topic = "occupancy";
+
+      // Calculate increment
+      sample.relativeMotion = incrementValue(
+        event.state.lastMotion,
+        sample.motion,
+      );
+      event.state.lastMotion = sample.motion;
+
+      if (sample.relativeMotion > 1) {
+        sample.occupancy = 2;
+        sample.occupied = true;
+      } else {
+        sample.occupancy = 0;
```

Comment:
only one motion count in the interval means pending/leaving? possibly add a comment about this assumption